File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Files/FileBasic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Objects.Files
{
    public class FileBasic : IFileData
    {
        public virtual string ID { get; set; }
        public virtual string Code { get; set; }
        public virtual string Name { get; set; }
        public virtual int Order { get; set; }
        public virtual string Description { get; set; }
        public virtual int State { get; set; }
        public virtual decimal Size { get; set; }
        public virtual string MIMEType { get; set; }
        public virtual FilePathType PathType { get; set; }
        public virtual string Path { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Files/FileBasic.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Files;



class FileBasic extends  PathTypes implements IFileData
{   public $MIMEType;
    public $PathType;//PathType enum sınıfı tipinde
    public $Path;
    public $Size;
    public $Details;
    public $Description;
    public $State;
    public $OrderNo;
    public $ID;

}